stories/components/radio.stories.tsx

```diff
@@ -29,15 +26,12 @@ const ExampleGroup = styled.div`
   gap: 20px;
 `;
 export function ControlBlueprint(
-  props: Omit<
-    BlueprintjsRadioGroupProps,
-    'onChange' | 'selectedValue' | 'children'
-  >,
+  props: Omit<RadioGroupProps, 'onChange' | 'selectedValue' | 'children'>,
 ) {
   const [option, setOption] = useState(options[2]);
```

Comment:
We can make this example simpler by using the `value` in the state, instead of the whole option.

Reply:
improved